Sources/Sentry/SentryTracer.m

```diff
@@ -96,6 +96,7 @@ @implementation SentryTracer {
     dispatch_block_t _idleTimeoutBlock;
     NSMutableArray<id<SentrySpan>> *_children;
     BOOL _startTimeChanged;
+    BOOL _timeout;
```

Comment:
`m`: I'm confused. We only set this to YES, and never use it. Or am I missing something?

Reply:
True. This was a first idea that I abandoned, after learning more about our messy tracer flow.
I will remove it